Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue5003.xaml.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 5003, "iOS StrikeThrough applied to null string throws error", PlatformAffected.iOS)]
+	public partial class Issue5003 : ContentPage
+	{
+		public Issue5003()
+		{
+			InitializeComponent();
```

Comment:
you need to wrap this on #if APP so it doesn't run on UITest project

Reply:
Done!